src/core_cli/Core_command.ml

```diff
@@ -189,9 +189,8 @@ let semgrep_core_with_one_pattern (config : Core_scan_config.t) : unit =
       let target_info, _skipped = Core_scan.targets_of_config config in
       let files =
         target_info
-        |> List_.map (function
-             | `CodeTarget (t : Input_to_core_t.code_target) -> t.path
-             | `LockfileTarget (t : Input_to_core_t.lockfile_target) -> t.path)
+        |> List_.map (fun (t : Target.t) ->
+               Target.internal_path_to_content t |> Fpath.to_string)
```

Comment:
would be easier to read as Target.path t |> Fpath.to_string.

Also we should probably use Fpath.t for files, so we don't beed Fpath.to_string.